.travis.yml

```diff
@@ -11,7 +11,7 @@ rvm:
   - 2.4.2
   - ruby-head
   - jruby-9.0.5.0
-  - jruby-9.1.15.0
+  - jruby-9.2.0.0
```

Comment:
should we remove `jruby-9.0.5.0` as well then?

Reply:
You'll probably not get a lot of benefit testing for it.

Reply:
I can drop that, in this PR.